ads-cert-examples/scala/src/main/scala/Main.scala

```diff
@@ -0,0 +1,88 @@
+import java.io.{FileInputStream, InputStreamReader}
+import java.nio.charset.StandardCharsets
+import java.security.spec.X509EncodedKeySpec
+import java.security._
+
+import org.apache.commons.codec.binary.Base64
+
+import org.bouncycastle.asn1.ASN1ObjectIdentifier
+import org.bouncycastle.asn1.x9.ECNamedCurveTable
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter
+import org.bouncycastle.openssl.{PEMKeyPair, PEMParser}
+
+import scala.util.{Failure, Success, Try}
+
+
+object Main {
+
+  Security.addProvider(new BouncyCastleProvider())
+
+  def main(args: Array[String]): Unit = {
+    val message = Seq(
+      "2824f5d3-a2dc-49ec-8358-dbe92bd01ec2", // source.tid
+      "a:180720yz:1807241326", // ad.pcv
+      "newsite.com", // site.domain
+      "", // app.bundle
+      "", // user.consent
+      "vd", // placement
+      "192.168.1.1", // device.ip
+      "", // device.ipv6
+      "", // device.ifa
+      "", // device.ua
+      "" // ad.video player size?
+    ).mkString(":")
+
+    val tryResult = for {
+      (privateKey, publicKey) <- loadKeys(args)
+      signature <- sign(privateKey, Signature.getInstance("SHA256withECDSA"), message)
+      _ <- Try(println(s"publisher signature: ${Base64.encodeBase64String(signature)}"))
```

Comment:
Should be able to get by without lifting this into the context of a `Try`:

```scala
_ = println(s"publisher signature: ${Base64.encodeBase64String(signature)}")
```

Note the change of the `<-` to a `=`.